bin/upgrading/update-db-general.pl

```diff
@@ -198,6 +198,23 @@
 )
 EOC
 
+register_tablecreate("faqhist", <<'EOC');
+CREATE TABLE faqhist (
+    rev mediumint(8) unsigned NOT NULL auto_increment,
+    faqid mediumint(8) unsigned NOT NULL,
+    question text,
+    summary text,
+    answer text,
+    sortorder int(11) default NULL,
+    faqcat varchar(20) default NULL,
+    lastmodtime datetime default NULL,
```

Comment:
The code to use `datetime` and `NOW()` rather than `INT UNSIGNED` and `UNIX_TIMESTAMP()` was taken from the existing code for the faq table. I'd prefer to keep the timestamps used by the new faqhist table in the same format as those used for the existing faq table.